ui/src/app/applications/components/application-retry-options/application-retry-options.scss

```diff
@@ -1,3 +1,10 @@
+.application-retry-options-wrap {
```

Comment:
Nitpick: we are using https://css-tricks.com/bem-101/ to avoid CSS conflcts. It is not perfect but the best we've got so far. For concistency can you rename `application-retry-options-wrap` to `application-retry-options`? e.g. 

```

.application-retry-options {
    margin-bottom: 1em;
    label {
        font-size: 15px;
    }

    .application-retry-options__item {
        margin: 12px 0;
        padding-top: 12px;
        position: relative;
        border: none;
    }
}
```

Reply:
@alexmt thanks 🙏 
I pushed fix